src/Router.js

```diff
@@ -307,7 +306,7 @@ var Router = React.createClass({
   },
 
   _renderScene(route, navigator) {
-    return cloneElement(route.component, route.props);
+    return createElement(route.component, route.props);
```

Comment:
Added check if object can be cloned. As i understood it fires undefined exception when cloned object is not and instance of Component class